src/core/core.datasetController.js

```diff
@@ -588,7 +582,7 @@ helpers.extend(DatasetController.prototype, {
 		var stacked = canStack && meta._stacked;
 		var indices = getSortedDatasetIndices(chart, true);
 		var otherScale = this._getOtherScale(scale);
-		var otherScaleBounds = getUserBounds(otherScale);
+		var {otherMin, otherMax} = getUserBounds(otherScale);
```

Comment:
doesn't this ned to be 

```javascript
var {min: otherMin, max:otherMax} = getUserBounds(otherScale);
```

Reply:
Yes, fixed and added a simple test to catch the error